update Contributing to Docs on Alert and Expandable components (#12485)
getsentry/sentry-docs: docs/contributing/pages/components.mdx

```markdown {tabTitle:No Title}
<Alert>
This is an info alert without a title.
</Alert>
```

Attributes:

- `title` (string) - optional
- `level` (string: `'info' | 'warning' | 'success'`) - optional, defaults to `'info'`

> 

Reviewer: ```suggestion
**Where to place alerts:**
- Make sure your alerts aren't breaking the flow of the content. For example, don't put an alert in between two paragraphs that explain a single concept. Use alerts sparingly; too many alerts on a single page can be overwhelming.
```

**When to use which alert level:**

- **Info**: Use this to emphasize information that users should read. For example:
  - information needed for users to succeed
  - information you'd typically wrap in a note callout
- **Warning**: Use this to alert users to potential risks they should be aware of. For example:

>   - warnings about potential errors, e.g., caused by common oversights in project configuration

Reviewer: ```suggestion
  - warnings about potential errors, such as ones caused by common oversights in project configuration
```

  - warnings regarding breaking changes

  Best practices:

  - always use a title with this alert level

- **Success**: Use this to provide optional information that can help users be more successful. For example:

>   - information that helps improve an already successful task, e.g., link to additional reading material

Reviewer: ```suggestion
  - information that helps improve an already successful task, such as a link to additional reading material
```

## Expandable

Render an expandable section to provide additional information to users on demand.

<Expandable title="Here's something worth noting">
  This is an expandable section in an `'info'` alert style. Use this to provide

>   additional information that is good to know.

Reviewer: ```suggestion
  additional information that is helpful, but not crucial.
```

</Expandable>

<Expandable permalink title="With permalink">
  This is an expandable section in an `'info'` alert style with a title wrapped
  in a link.
</Expandable>

<Expandable title="This could be important to some users" level="warning">
  This is an expandable section in a `'warning'` alert style. Use this to warn

>   users about potential risks.

Reviewer: ```suggestion
  users about minor potential risks.
```

</Expandable>

<Expandable title="Want some tips to improve something?" level="success">
  This is an expandable section in a `'success'` alert style. Use this to
  provide optional information that can help users be more successful.
</Expandable>

```markdown {tabTitle:Example}
<Expandable title="Here's something worth noting">
  This is an expandable section in an `'info'` alert style.
</Expandable>
```

```markdown {tabTitle:Permalink}
<Expandable permalink title="With permalink">
  This is an expandable section with a permalink.
</Expandable>
```

```markdown {tabTitle:Warning}
<Expandable level="warning" title="This could be important to some users">
  This is an expandable section in a `'warning'` alert style.
</Expandable>
```

```markdown {tabTitle:Success}
<Expandable level="success" title="Want some tips to improve something?">
  This is an expandable section in a `'success'` alert style.
</Expandable>
```

Attributes:

- `title` (string)
- `permalink` (boolean) - optional: wraps the title in a link and shows it in the table of contents.

**When to use expandables:**

> Expandables help keep our documentation clutter-free, allowing users to focus on important information while letting others explore additional relevant details.

Reviewer: ```suggestion
Expandables help keep our documentation clutter-free, allowing users to focus on important information while providing them the option to explore additional relevant details.
```
